src/EffectID.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+// tolua_begin
+enum ENUM_EFFECT_ID
```

Comment:
Use a typed enum class to avoid undefined behavior if we want to allow plugins to pass other values, and to reduce the pollution of the global namespace.